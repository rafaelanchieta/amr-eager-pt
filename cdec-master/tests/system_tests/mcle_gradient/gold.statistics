-lm_nodes 4
-lm_edges 8
-lm_paths 5
constr_nodes 5
constr_edges 6
constr_paths 2
-lm_nodes 4
-lm_edges 8
-lm_paths 5
constr_nodes 5
constr_edges 5
constr_paths 1
-lm_nodes 7
-lm_edges 17
-lm_paths 15
constr_nodes 8
constr_edges 13
constr_paths 6
