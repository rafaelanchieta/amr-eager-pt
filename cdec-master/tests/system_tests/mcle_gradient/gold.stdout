0	**OBJ**=3.05441;RewriteXY=(-)-2.0907 Top=(-)-inf SomeFeat=-0.840281 F1=-1.96646 F2=-2.82869 F3=(-)-2.14693 F4=-2.23785 F5=-2.51189 F6=(-)-1.35609 F7=(-)-3.79473 F8=-2.12996 F9=(-)-0.60656 F10=-0.74837 F11=-4.03796 F12=(-)-1.55837 F13=-4.11699
